packages/shadydom/src/patch-events.js

```diff
@@ -271,6 +272,29 @@ let EventPatches = {
   },
 };
 
+const hasDescriptors = utils.settings.hasDescriptors;
+
+const eventPhaseDescriptor =
+  hasDescriptors &&
+  Object.getOwnPropertyDescriptor(Event.prototype, 'eventPhase');
+if (eventPhaseDescriptor) {
```

Comment:
Why is this conditional? It's supported back to IE9. Add a comment?

Reply:
Added comment. Chrome 41 (a test browser) is missing it.

Reply:
Are we still testing on 41?

Which supported browser does that stand in for now... older Cobalt?

Reply:
Actually no, but it's probably good to continue to test on given the range of browsers needed... internally.